README.md

```diff
@@ -1,175 +1,302 @@
-        New python executable in usaspending-api/bin/python3.5
-        Also creating executable in usaspending-api/bin/python
-        Installing setuptools, pip...done.
 
-        (usaspending-api)$
+Now we will allow `venv` to manage the virtual environment by the following:
+```
+$ echo "layout python-venv python3.5" >> usaspending-api/.envrc
+$ cd usaspending-api
+$ direnv allow
+$ source activate
+```
 
-7. This new environment will be active until you run the `deactivate` command. You can re-activate the environment again at any time by typing `workon usaspending-api`.
+The `source activate` command will open the virtual environment for usaspending-api. Anything you pip install from this point forward will be installed into the *usaspending-api* environment rather than your machine's global Python environment. Your command line prompt indicates which (if any) virtual environment is active.
 
-### USAspending API Backend
+  
 
-Decide where on your machine you want the USAspending API code to live. From the command line, navigate there and clone the USAspending API repository from GitHub to your local environment:
+*Remember to use `source activate` as you work in the project folder; You can exit the virtual environment by using `deactivate`*
 
-        $ git clone https://github.com/fedspendingtransparency/usaspending-api.git
+  
 
-Navigate to the USAspending API's main folder:
+## Install the dependencies
+Use `source activate` to open the virtual environment
+```
+$ source activate
+(python-venv-3.5.0) $ pip install -r requirements/requirements.txt
+```
 
-        $ cd usaspending-api
+  
 
+Install caching dependencies. (This is optional, and should only be done if you have installed Memcached and require AWS caching for development)
 
-Install the dependencies.
+```
+$ pip install -r requirements/caching_requirements.txt
+```
 
-        $ pip install -r requirements/requirements.txt
+  
 
-Install caching dependencies. (This is optional, and should only be done if you have installed Memcached and require AWS caching for development)
+Next, configure your local settings. The `settings.py` file will check for an environment variable named `DATABASE_URL`.
 
-        $ pip install -r requirements/caching_requirements.txt
+  
 
-Next, configure your local settings. The `settings.py` file will check for an environment variable named `DATABASE_URL`. You can export this variable on the command line, or use a library like [autoenv](https://github.com/kennethreitz/autoenv) to load your environment variables when you `cd` into that directory.
+## Set your virtual environment export commands
 
-Exporting your `DATABASE_URL` environment variable should look something like this:
+Export your `DATABASE_URL` environment variable by appending the following to your `.envrc` file
 
-        $ export DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
+```
+$ export DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
```

Comment:
This doesn't work for Windows users does it? Either all instructions should have the necessary details for the three envs or we drop all Windows steps and just provide MacOS/Linux instructors.

I don't know if the Ubuntu integration with Windows 10 might work as a middle ground.